apps/challenges/models.py

```diff
@@ -12,6 +14,11 @@
 class Challenge(TimeStampedModel):
 
     """Model representing a hosted Challenge"""
+
+    def __init__(self, *args, **kwargs):
+        super(Challenge, self).__init__(*args, **kwargs)
+        self.__original_evaluation_script = self.evaluation_script
```

Comment:
Yeah, let's follow the convention that we have been following i.e use single underscores.